packages/api/test/__fixtures__/sdk/petstore/index.ts

```diff
@@ -0,0 +1,617 @@
+import Oas from 'oas';
+import APICore from 'api/core';
+import definition from './petstore.json';
+
+export default class SDK {
+  spec: Oas;
+  core: APICore;
+  authKeys: (number | string)[][] = [];
+
+  constructor() {
+    this.spec = Oas.init(definition);
+    this.core = new APICore(this.spec, 'api/1.0.0');
+  }
+
+  /**
+   * Optionally configure various options, such as response parsing, that the SDK allows.
+   *
+   * @param config Object of supported SDK options and toggles.
+   * @param config.parseResponse If responses are parsed according to its `Content-Type` header.
+   */
+  config(config: ConfigOptions) {
+    this.core.setConfig(config);
+  }
+
+  /**
+   * If the API you're using requires authentication you can supply the required credentials
```

Comment:
It would be nice to have this comment auth aware, so it told you how to use it instead of a generic message (assuming I'm reading this file right as the output of the codegen)